src/models/planet.js

```diff
@@ -7,13 +7,16 @@ const planetSchema = new mongoose.Schema({
 	name: {
 		type: String
 	},
+	tier: {
+		type: Number
+	},
 	point: {
 		type: Number
 	},
-	characteristic_ids: [{
+	travel_exprense: {
```

Comment:
What is it?
travel_experience? or travel expense?
if it's travel expense. I think **travel_cost** get more understanding